### ISSUE TITLE
A11y - QList does not have role="list" while QItem does have role="listitem"

### ISSUE THREAD

**Issue by Evertvdw** [quasarframework/quasar#17439]
### What happened?

The QList does not have an associated `role="list"` attribute, while the child items do have `role="listitem"`. This can be added manually but the `role="listitem"` on QItem is hardcoded and cannot be removed so the default behavior now is not consistent with A11y guidelines.

### What did you expect to happen?

QList should have the correct role

### Reproduction URL

https://codepen.io/Mett-codepens/pen/PorEaeR?editors=101

### How to reproduce?

Inspect the list and see there is no `role="list"`.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar), Accessibility [a11y] (quasar)

### Platforms/Browsers

Firefox, Chrome, Safari, Microsoft Edge, iOS, Android, Electron
